blocks/section/section.css

```diff
@@ -0,0 +1,7 @@
+.section{
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+  line-height: 1.75rem;
```

Comment:
Hey! I think we can remove `line-height: 1.75rem;` here since we don't write in this file any other styles for text. It is also never used because we apply `line-height` again for `section__intro`.

Reply:
@MarPostovik done✅